src/platform/qt/Window.cpp

```diff
@@ -1031,17 +1031,23 @@ void Window::updateTitle(float fps) {
 		NoIntroGame game{};
 		uint32_t crc32 = 0;
 		m_controller->thread()->core->checksum(m_controller->thread()->core, &crc32, CHECKSUM_CRC32);
-
-		char gameTitle[17] = { '\0' };
+		
 		mCore* core = m_controller->thread()->core;
```

Comment:
Move this above the checksum and then change it to use `core`. (Yes, it was like this before. However, this is a fix I'd like and since the code around it is being modified now is a good time to fix it.)